unicodetools/data/emoji/dev/internal/emojiOrdering.txt

```diff
@@ -137,214 +135,14 @@
 👭
 👫
 👬
-💏 
```

Comment:
Something funny is going on with the ordering. 
For the eventual ICU collation rules, normally we need to have **all** combinations of modifiers that are applicable to a human ZWJ sequence. There is one exception: any final modifier can be left off.

Now, the EmojiOrdering class has some logic built in to avoid listing all the sequences, but I suspect that it doesn't handle these cases properly, which is why they are explicitly listed. So their removal makes me nervous.

Reply:
None of these sequences were in https://www.unicode.org/emoji/charts-14.0/emoji-ordering-rules.txt, and they continue to not be in https://www.unicode.org/emoji/charts-15.0/emoji-ordering-rules.txt… since this file being in dev/ is only causing consternation, can it safely be removed?

Reply:
unicodetools/data/... is the normal place for data files. As noted in #283 that's where we would expect the one file to live.

Ned had said "We have #283 for cleaning up the duplicated files later." -- that would be ok with me. That is, keeping duplicates in this PR (with the same contents) and deduplicating later.